### ISSUE TITLE
On-map report not completing

### ISSUE THREAD

**Issue by ODOT-RPTD-sh** [ODOT-PTS/TNExT#220]
Cannot get the on-map report to generate (both Staging and Production).  Loading wheel does not finish.  (Both on the ODOT network and off) - Chris Erickson.  This is the report that generates a report based on lat/long centroid location and Census data.
![on-map Capture](https://user-images.githubusercontent.com/49998980/70465886-cf9f2d00-1a76-11ea-8cd5-8d48aaffa3e3.PNG)

**Comment by thomastrillium** [comment]
@anniekfifer will investigate, @ODOT-RPTD-mb believes there's a software dependency that changed

**Comment by anniekfifer** [comment]
https://docs.geotools.org/latest/userguide/welcome/upgrade.html

**Comment by anniekfifer** [comment]
While working on this I discovered that the docker image was defaulting to java 11 -- the dockerfile installs jdk 8 but never sets it as the java version.

As far as I can tell (based on the readme, installation of jdk 8 and lack of mention of updating java in github) this was unintentional, and java 8 is supposed to be used. I fixed this in the dockerfile, along with updating the geotools repos per the guide above (note that downgrading java seemed to be necessary to fix this issue in a reasonable amount of time, updating the geotools repos alone led me down a rabbit hole of updating/adding/removing dependencies that I never got resolved)

@ODOT-RPTD-mb @ODOT-RPTD-sh FYI testing should include a good amount of "smoke testing" since this ended up being a pretty major change. I think a good way for me to indicate that smoke testing is needed is to add a card to to the "Development Done (Ready to QA)" column, so I've done that for this: https://github.com/ODOT-PTS/TNExT/projects/5. This is in addition to testing the on-map report specifically.
